conf/inter_athena.conf

```diff
@@ -153,6 +153,9 @@ roulette_table: db_roulette
 // Use SQL item_db, mob_db and mob_skill_db for the map server? (yes/no)
 use_sql_db: no
 
+// Use SQL item_cash_db, item_cash_db2 for the map server? (yes/no)
+use_sql_cashdb: yes
```

Comment:
Maybe it's better if it is **no** by default? I mean, up until now it was one with the item_db so by default it would've been **no**. Besides, wouldn't this require rAthena users to import an additional SQL file to be able to run a fresh server?

Reply:
yes it is better to set no by default, sorry forgot to change that as well.